### PR TITLE
Feature/update grouped images

### PR DESCRIPTION
**Pull request by capetillo**
### DESCRIPTION
Background:
In our previous update on the ThumbnailRow component, we introduced a feature to group images by their smart stack, aiming to enhance user experience by minimizing the clutter and displaying only the most recent image from each stack. However, @wrosing noticed that the grouped images were not updating in real time as new data came in, only after refreshing the page.

Implementation:
To tackle this, I have made adjustments so that as new image data comes in, the grouped images update accordingly. Please note that this fix is speculative, as testing is pending. However, due to Wayne's requirements, we are deploying it for live testing and feedback. 

Code:
The current implementation can be found in images.js. In this file, inside the `update_new_image` action `group_images` is dispatched, allowing for the grouped_images object to listen for new data and update the respective image stacks in real time. 

Fixed Features:
Real-time Image Update:

1. The images within each group should dynamically update, ensuring the most recent images are always displayed, reflecting the real time data.

Note:
This feature is pushed for immediate deployment with the understanding that it has not been tested. There might be discrepancies in displaying the appropriate images corresponding to each site. Please verify the _basefile_name_ located above the image aligns with _selected site_ on the left. Please report any discrepancies or anomalies observed for correction.

### VISUALS

**basefile_name and selected site matching**
Screenshot 2023-09-21 at 10 42 32 AM

**basefile_name and selected site not matching**
Screenshot 2023-09-21 at 10 49 04 AM

### CODE REVIEW

**Comment by markBowman** [comment]
Looks good. This PR is behind the dev branch - the extra commits on dev should be merged with the PR prior to merging this PR to dev

File: src/components/projects/CreateProjectForm.vue [hunk 1]
```diff
@@ -820,8 +820,8 @@ export default {
       max_fits_header_length: 68,
       generic_filter_list: ['Lum', 'Red', 'Green', 'Blue', 'HA', 'O3', 'S2', 'EXO'],
       generic_camera_areas: [
-        '600%', '500%', '400%', '300%', '220%', '133%',
```
Comment (markBowman) [inline src/components/projects/CreateProjectForm.vue:823]: Did you mean for this change to be in this PR?

Comment (capetillo) [reply src/components/projects/CreateProjectForm.vue:823]: I thought I reverted the changes. Let me fix!

Comment (capetillo) [reply src/components/projects/CreateProjectForm.vue]: fixed back to what it was!

**Review by markBowman — APPROVED** [review]
lgtm

**Review by markBowman — APPROVED** [review]
Seems reasonable. I hope it works as intended.
Fantastic PR description.

**Pull request merged**